bounce.py

```diff
@@ -390,6 +394,19 @@ def _button_release_cb(self, win, event):
                     self._start_step()
         return True
 
+    def _mouse_motion_cb(self, widget, event):
+        if self.ball.grabbed and self._fraction != None:
+            if event.is_hint:
+                mouse_x, mouse_y, state = event.window.get_pointer()
+            else:
+                mouse_x = event.x
+                mouse_y = event.y
```

Comment:
`bounce.py:403:17: F841 local variable 'mouse_y' is assigned to but never used`

You haven't thought this through?  :grin: